addons/l10n_in/data/template/account.fiscal.position-in.csv

```diff
@@ -11,18 +12,6 @@
 "","","sgst_purchase_12","igst_purchase_12"
 "","","sgst_purchase_18","igst_purchase_18"
 "","","sgst_purchase_28","igst_purchase_28"
-"fiscal_position_in_export","Export","sgst_sale_1","igst_sale_1"
```

Comment:
Should not this be mapped to a 0% instead?

Reply:
@jva-odoo should we do it or not?

Reply:
No, we need it new tax that effect is 0 like 18% GST but has a repartition line of +100% and -100%
It also depends on Customer holds a Letter of Undertaking(LUT) or not.
so no automation for that